sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -462,6 +462,57 @@ object Union {
   }
 }
 
+abstract class UnionBase extends LogicalPlan {
```

Comment:
can we have a descriptive comment for this operator saying what it represents, when it is constructed, examples, etc.?